client/src/containers/App/routes/private.routes.js

```diff
@@ -3,9 +3,6 @@ import { Route, Redirect, Switch } from 'react-router-dom';
 import { connect } from 'react-redux';
```

Comment:
remove connect function from this file also remove mapStateToProps from here which is on line 37. Reason for this because we are not using state props anywhere in this file.

Reply:
@taleymirza this feedback is remaining

Reply:
We need userstate to determine first if the user is logged in to route him to editProfile page. So we will need connect and mapStateToProps